File: C++ ad y ad/3_ Orientado a objetos/S02.1_ Paracaidista/Solucion_ejercicio_paracaidista/Ejercicio_paracaidista/paracaidista.cpp
#include "paracaidista.h"


Paracaidista::Paracaidista()
{
	nombre = "";
	edad = 0;
	terrenoDondeCaer = "";
}
Paracaidista::~Paracaidista()
{
	cout << "El paracaidista " << nombre << " ha sido liberado de la memoria.\n\n";
}


string Paracaidista::getNombre()
{
	return nombre;
}
int Paracaidista::getEdad()
{
	return edad;
}
string Paracaidista::getTerrenoDondeCaer()
{
	return terrenoDondeCaer;
}

void Paracaidista::setNombre(string nombre)
{
	this->nombre = nombre;
}
void Paracaidista::setEdad(int edad)
{
	this->edad = edad;
}
void Paracaidista::setTerrenoDondeCaer(string terrenoDondeCaer)
{
	this->terrenoDondeCaer = terrenoDondeCaer;
}

File: C# ad y ad/2_ Orientado a objetos/S02.1_ Paracaidista/Solucion_ejercicio_paracaidista/Ejercicio_paracaidista/Paracaidista.cs

namespace Ejercicio_paracaidista
{
	class Paracaidista
	{
		string nombre;
		int edad;
		string terrenoDondeCaer;


		public Paracaidista()
		{
			nombre = "";
			edad = 0;
			terrenoDondeCaer = "";
		}


		public string GetNombre()
		{
			return nombre;
		}
		public int GetEdad()
		{
			return edad;
		}
		public string GetTerrenoDondeCaer()
		{
			return terrenoDondeCaer;
		}

		public void SetNombre(string nombre)
		{
			this.nombre = nombre;
		}
		public void SetEdad(int edad)
		{
			this.edad = edad;
		}
		public void SetTerrenoDondeCaer(string terrenoDondeCaer)
		{
			this.terrenoDondeCaer = terrenoDondeCaer;
		}
	}
}
